RaveBusinessLogic/TauDEM.xml

```diff
@@ -9,58 +9,48 @@
-                                <Children>
-                                    <Node label="Pitfill" xpath="Raster[@id='PITFILL']" type="raster" symbology="dem" transparency="40" />
-                                    <Node label="Flow Directions (D∞)" xpath="Raster[@id='DINFFLOWDIR_ANG']" type="raster" symbology="dinfflowdir_ang" transparency="40" />
-                                    <Node label="Contributing Area (D∞)" xpath="Raster[@id='AREADINF_SCA']" type="raster" symbology="areadinf_sca" transparency="40" />
+                                <Children collapsed="true">
+                                    <Node label="Pitfill DEM" xpath="Intermediates/Raster[@id='PITFILL']" type="raster" symbology="dem" transparency="40" />
+                                    <Node label="Flow Directions (D∞)" xpath="Intermediates/Raster[@id='DINFFLOWDIR_ANG']" type="raster" symbology="dinfflowdir_ang" transparency="40" />
+                                    <Node label="Contributing Area (D∞)" xpath="Intermediates/Raster[@id='AREADINF_SCA']" type="raster" symbology="areadinf_sca" transparency="40" />
                                 </Children>
                             </Node>
                             <Node label="Specialized Grid Analysis">
-                                <Children>
-                                    <Node label="TWI" xpath="Raster[@id='TWI_RASTER']" type="raster" symbology="twi" transparency="40" />
-                                    <Node label="HAND" xpath="Raster[@id='HAND_RASTER']" type="raster" symbology="hand" transparency="40" />
-                                </Children>
-                            </Node>
-                            <Node label="TauDEM Outputs">
-                                <Children>
-                                    <Node label="D∞ Slope" xpath="Raster[@id='DINFFLOWDIR_SLP']" type="raster" symbology="dinfflowdir_slp" transparency="40" />
-                                    <Node label="DEM Hillshade" xpath="Raster[@id='HILLSHADE']" type="raster" symbology="hillshade" id="hillshade" />
+                                <Children collapsed="false">
+                                    <Node label="TWI" xpath="Outputs/Raster[@id='TWI_RASTER']" type="raster" symbology="twi" transparency="40" />
+                                    <Node label="HAND" xpath="Outputs/Raster[@id='HAND_RASTER']" type="raster" symbology="hand" transparency="40" />
                                 </Children>
                             </Node>
                         </Children>
+                    </Node>
                     <Node label="GDAL Outputs">
                         <Children>
-                            <Node label="Slope Analysis (D8)" xpath="" type="raster" symbology="" transparency="" id="d8_slope" />
+                            <!-- Node label="Slope Analysis (D8)" xpath="" type="raster" symbology="" transparency="" id="d8_slope" /> -->
```

Comment:
This should have the `symbology="slope"` from "shared" (e.g. [this one](https://github.com/Riverscapes/RiverscapesXML/blob/master/Symbology/qgis/Shared/slope.qml)) because this one is the slope degrees one.